packages/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -116,6 +116,8 @@ MeetingUtil.joinMeeting = (meeting, options) => {
 };
 
 MeetingUtil.cleanUp = (meeting) => {
+  meeting.ensureLLMConnected();
```

Comment:
should this call maybe be part of the promise chain returned by this function? Otherwise cleanUp() may resolve before LLM is disconnected

Reply:
I've changed this to be part of the promise chain